library/src/comparable.ts

```diff
@@ -2,9 +2,7 @@ import { expect } from 'vitest';
 
 /**
  * Prepares an object so that it can be compared to another with `.toEqual`.
- *
```

Comment:
Can we disable the rule that does not allow empty lines to keep them? For me it makes it more readable.

Reply:
Thank you!